apps/cookbook/src/app/examples/badge-example/badge-example.component.html

```diff
@@ -1,29 +1,16 @@
 <h2>Number</h2>
```

Comment:
Consider wrapping all of this in a card or container, and give it a little spacing so the popout example gets a little more spacing to the left.

Reply:
Beautiful!!